src/crosstargeting_override.props.sample

```diff
@@ -1,37 +1,63 @@
-			The packages are located under this directory: "%USERPROFILE%\.nuget\packages".
-		-->
-		<!-- <NugetOverrideVersion>1.0.0-dev.1</NugetOverrideVersion> -->
-
-		<!--
-			This property is used to control the platforms compiled by Visual Studio, and
-			allows for a faster build when testing for a single platform.
-
-				Instructions:
-				1) Copy this file and remove the ".sample" name
-				2) Adjust the NugetOverrideVersion property below
-				3) Make sure to do a Rebuild, so that nuget restores the proper packages for the new target
-
-			Available build targets
-
-				uap10.0.18362 (Windows)
-				net6.0-windows10.0.18362 (WinAppSDK Windows)
-				xamarinios10 (iOS)
-				MonoAndroid12.0 (Android 12.0)
-				netstandard2.0 (WebAssembly, Skia)
-				net6.0-ios (.NET 6 iOS)
-				net6.0-android (.NET 6 Android)
-				net6.0-maccatalyst (.NET 6 macOS Catalyst)
-				net6.0-macos (.NET 6 macOS AppKit)
-				xamarinmac20 (macOS)
-
-			Include netstandard2.0 to make sure all projects build
-		-->
-		<!-- <UnoTargetFrameworkOverride>netstandard2.0;uap10.0.18362</UnoTargetFrameworkOverride> -->
+		<!-- See documentation for each property below -->
+
+		<!-- Include netstandard2.0 to make sure all projects build -->
+		<!--<TargetFrameworkOverride>netstandard2.0;net6.0-android</TargetFrameworkOverride>-->
+		<!--<NugetOverrideVersion>2.6.0-dev.34</NugetOverrideVersion>-->
+		<!--<DisableNet6MobileTargets>true</DisableNet6MobileTargets>-->
+
+		<!-- 
+		
+	  ### TargetFrameworkOverride ###
+
+		This property controls the platform built by Visual Studio.
+
+		Available build targets:
+
+		┌───────────────────────────────────────────┬─────────────────────────┐
+		│ TargetFrameworkOverride                   │ Platform                │
+		├───────────────────────────────────────────┼─────────────────────────┤
+		│ uap10.0.18362                             │ Windows                 │
+		│ net6.0-windows10.0.18362                  │ WinAppSDK Windows       │
+		│ xamarinios10                              │ iOS                     │
+		│ MonoAndroid12.0                           │ Android 12.0            │
+		│ netstandard2.0                            │ WebAssembly, Skia       │
+		│ net6.0-ios                                │ .NET 6 iOS              │
+		│ net6.0-android                            │ .NET 6 Android          │
+		│ net6.0-maccatalyst                        │ .NET 6 macOS Catalyst   │
+		│ net6.0-macos                              │ .NET 6 macOS AppKit     │
+		│ xamarinmac20                              │ macOS                   │
+		└───────────────────────────────────────────┴─────────────────────────┘
+
+		*** WARNING ***
+		Note that changing that property while the solution is opened leads to
+		unstable nuget restore operations, and Visual Studio instabilities such
+		as caching issues or crashes.
+
+		Always close the solution before changing or activating this property.
+		*** WARNING ***
+
+	  ### NugetOverrideVersion ###
```

Comment:
just inline the description between commented example, like we had in uno.ui: 
```xml
<!--
	MyProperty blah blah...
-->
<!--<MyProperty>1<MyProperty>-->
```
so people dont have to scroll up and down back and forth

Reply:
after reviewing the entire file, i think it is fine as is now...
not that people need to constantly to refer to the associated doc
it is mostly for the 1st few times

so having them all lump together at the top is probably preferred

Reply:
that said though, having all the examples ready to uncomment for `TargetFrameworkOverride` is still desired